tests/integration/internal/k8s/pods.go

```diff
@@ -81,6 +81,14 @@ func WaitUntilSumologicMockAvailable(
 	waitDuration time.Duration,
 	tickDuration time.Duration,
 ) {
+	if ctxopts.Namespace(ctx) == internal.AdditionalSumologicMockNamespace {
+		kubectlOpts := *ctxopts.KubectlOptions(ctx)
+		kubectlOpts.Namespace = internal.AdditionalSumologicMockNamespace
```

Comment:
Is this really necessary? The values of `ctxopts.Namespace(ctx)` and `kubectlOpts.Namespace` should be the same.

Reply:
It doesn't work without this part. We are waiting for the service which is not a part of test